### PR TITLE
[fix] quick fix of executed trees mismatch

### PR DESCRIPTION
**Pull request by lightmark**
### Description

There is an abvious mismatch error happening. But not sure whether there are more problems involved.

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2597)

### CODE REVIEW

File: storage/aptosdb/src/lib.rs [hunk 1]
```diff
@@ -1201,7 +1201,7 @@ impl DbReader for AptosDB {
                                 None
                             )
                         } else {
-                            let num_txns = committed_version + 1;
+                            let num_txns = executed_trees.state().base_version.map_or(0, |v| v + 1);
```
Comment (zekun000) [inline storage/aptosdb/src/lib.rs:1204]: isn't get_frozen_subtree_hashes going to fail if this exceeds the committed version?

Comment (lightmark) [reply storage/aptosdb/src/lib.rs:1204]: This is the real committed_version, the old `committed_version` should be LI version.

Comment (zekun000) [reply storage/aptosdb/src/lib.rs:1204]: LI version is the committed version, i guess you mean this is the synced version

Comment (lightmark) [reply storage/aptosdb/src/lib.rs:1204]: By LI version I mean the version of the latest persisted LI. This line's version is the latest snapshot version, and now we have the latest txn_info version (synced_version). 
get_frozen_subtree_hashes has 0..synced_version >= latest_snapshot_version.

**Pull request closed without merging**